<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="tripleStoreChooserLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Triple Store:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tripleStoreChooser">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addTripleStoreButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new TripleStore</string>
   </property>
  </widget>
  <widget class="QLabel" name="tripleStoreLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Triple Store URL:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tripleStoreEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tripleStoreNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Triple Store Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tripleStoreNameEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
